ci/docker/python-wheel-windows-vs2017.dockerfile

```diff
@@ -81,7 +81,8 @@ ARG python=3.8
 RUN (if "%python%"=="3.7" setx PYTHON_VERSION "3.7.9" && setx PATH "%PATH%;C:\Python37;C:\Python37\Scripts") & \
     (if "%python%"=="3.8" setx PYTHON_VERSION "3.8.10" && setx PATH "%PATH%;C:\Python38;C:\Python38\Scripts") & \
     (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.7" && setx PATH "%PATH%;C:\Python39;C:\Python39\Scripts") & \
-    (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.2" && setx PATH "%PATH%;C:\Python310;C:\Python310\Scripts")
+    (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.2" && setx PATH "%PATH%;C:\Python310;C:\Python310\Scripts") & \
```

Comment:
Can we also bump make these releases up to date? ("3.10.8" instead of "3.10.2", etc.)
See https://www.python.org/downloads/ for the current list

Reply:
Should we update the others too? "3.9.7" --> "3.9.15", etcetera?

Reply:
Yes, that was my assumption.